test/vtcomposite-internationalize.test.js

```diff
@@ -0,0 +1,255 @@
+const test = require('tape');
+const internationalize = require('../lib/index.js').internationalize;
+const fs = require('fs');
+const path = require('path');
+const zlib = require('zlib');
+const mvtFixtures = require('@mapbox/mvt-fixtures');
+const vtinfo = require('./test-utils.js').vtinfo;
+const vt1infoValid = require('./test-utils.js').vt1infoValid;
+const tilebelt = require('@mapbox/tilebelt');
+
+
+test('[internationalize] success: buffer size stays the same when no changes needed', function(assert) {
+  const singlePointBuffer = mvtFixtures.get('002').buffer;
+
+  internationalize(singlePointBuffer, 'piglatin', {}, (err, vtBuffer) => {
+    assert.notOk(err);
+    assert.equal(vtBuffer.length - 3, singlePointBuffer.length, 'same size (with expected 3 byte difference due to explicit 4096 extent in output)');
+    assert.end();
+  });
+});
+
+test('[internationalize] success: single gzipped VT', function(assert) {
+  const singlePointBuffer = mvtFixtures.get('002').buffer;
+  const gzipped_buffer = zlib.gzipSync(singlePointBuffer);
+  internationalize(gzipped_buffer, 'piglatin', {}, (err, vtBuffer) => {
+    assert.notOk(err);
+    assert.equal(vtBuffer.length - 3, zlib.gunzipSync(gzipped_buffer).length, 'same size (with expected 3 byte difference due to explicit 4096 extent in output)');
+    assert.end();
+  });
+});
+
+test('[internationalize] success: gzipped output', function(assert) {
+  const singlePointBuffer = mvtFixtures.get('002').buffer;
+  internationalize(singlePointBuffer, 'piglatin', {compress:true}, (err, vtBuffer) => {
+    assert.notOk(err);
+    assert.equal(zlib.gunzipSync(vtBuffer).length - 3, singlePointBuffer.length, 'same size (with expected 3 byte difference due to explicit 4096 extent in output)');
+    assert.end();
+  });
+});
```

Comment:
you've done 3/4 compression tests. may as well also check gzip in -> gzip out as well